pkg/field/validation.go

```diff
@@ -49,7 +49,7 @@ func Validate(c Configuration, v *viper.Viper) error {
 		case reflect.String:
 			isNonZero = v.GetString(f.FieldName) != ""
 		case reflect.Slice:
-			isNonZero = len(v.GetStringSlice(f.FieldName)) == 0
+			isNonZero = len(v.GetStringSlice(f.FieldName)) != 0
```

Comment:
Do we have a test we can update to validate this?

Reply:
added 2 tests